tests/unit/Mocks/TransactionListMock.php

```diff
@@ -0,0 +1,303 @@
+<?php
```

Comment:
O que acha de trabalharmos com arquivos JSON ao invés de usar classes diretamente para retornar os valores de mocks?

Reply:
E como pegariamos o conteúdo? Se for com file_get_contents achei que ficaria um syntax um pouco ruim, aí fiz assim pq achei que ficaria mais facil de entender.

Reply:
Pensei no file_get_contents mesmo. a principal vantagem é de não precisar ignorar esses arquivos PHP a mais no test coverage. O que acha?